docs/meshstack.kubernetes.index.md

```diff
@@ -3,16 +3,18 @@ id: meshstack.kubernetes.index
 title: Integration
 ---
 
+
 meshStack supports project creation, configuration, user management and SSO for Kubernetes clusters.
 Our Kubernetes integration is distribution-independent and is purely based on "vanilla" upstream Kubernetes services.
 Operators should be able to successfully integrate any Kubernetes distribution of their chosing as long as it has the
 required APIs and configuration options available.
+We also support Azure AAD authenticated users to get seamless access to the AKS (Azure Kubernetes) cluster.
```

Comment:
I can try to emphasize more AKS but I guess a matrix is not a good idea because we sell stuff we don't support yet. And putting the non-supported platforms in the public docs is probably not a good selling point. :D